assets/css/src/generic/_forms-select.scss

```diff
@@ -63,7 +59,7 @@ select {
 
 .mailpoet-form-react-select__single-value {
   color: $color-text !important;
-  max-width: 100%;
+  max-width: 97%; // leave space for narrow input to search another value
```

Comment:
Wouldn't `calc(100% - a-few-px)` be better than flexible `3%`?

Reply:
You're right, updated